masterlist.yaml

```
@@ -21851,3 +21851,9 @@ plugins:
     msg: [ *alreadyInOrFixedByBDSv3_0 ]
 
 ###### Anything new can go after this (If you're not sure where to put it) ######
+
+  - name: 'Omni-No-Useless-Papers-SSE.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/51468' ]
```

May as well add the closing forward slash to links while we're at it.

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/51468/' ]
```